scripts/votca_compare.in

```diff
@@ -0,0 +1,141 @@
+#! /usr/bin/env python
+#
+# Copyright 2009-2018 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+VERSION='@PROJECT_VERSION@ #TOOLS_GIT_ID#'
+import sys
+import os
+import getpass
+import argparse
+
+PROGTITLE = 'THE VOTCA::TOOLS FILE COMPARISON'
+PROGDESCR = 'COMPARES TWO FILES TO EACH OTHER'
+VOTCAHEADER = '''\
+==================================================
+========   VOTCA (http://www.votca.org)   ========
+==================================================
+
+{progtitle}
+
+please submit bugs to bugs@votca.org 
+votca_compare, version {version}
+
+'''.format(version=VERSION, progtitle=PROGTITLE)
+
+def xxquit(what=''):
+  if what != '':
+    cprint.Error("ERROR: {what}".format(what=what))
+  sys.exit(1)
+
+def RepresentsFloat(s):
+    try: 
+        float(s)
+        return True
+    except ValueError:
+        return False 
+
+# =============================================================================
+# PROGRAM OPTIONS
+# =============================================================================
+
+class ToolsHelpFormatter(argparse.HelpFormatter):
+  def _format_usage(self, usage, action, group, prefix):
+    return VOTCAHEADER
+    
+progargs = argparse.ArgumentParser(prog='votca_compare',
+    formatter_class=lambda prog: ToolsHelpFormatter(prog,max_help_position=70),
+  description=PROGDESCR)
+  
+progargs.add_argument('-f1', '--file1',
+    dest='file1',   
+    action='store',
+    required=True,
+    type=str,
+    help='File1 to compare.')
+
+progargs.add_argument('-f2', '--file2',
```

Comment:
For validation if the argument is a file, you could use `argparse.FileType` https://docs.python.org/2.7/library/argparse.html#filetype-objects

Reply:
thanks very helpful

Reply:
the rest of your comments are too, especially the linter thing.